From: Kasumi Hanazuki <kasumi@rollingapple.net>
Date: Mon, 16 Sep 2019 08:49:27 +0000
Subject: Include custom mrbgems

---
 build_config.rb | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/build_config.rb b/build_config.rb
index b543ff5..df990a2 100644
--- a/build_config.rb
+++ b/build_config.rb
@@ -13,6 +13,9 @@ def gem_config(conf)
   conf.gem github: 'k0kubun/mruby-tempfile',    checksum_hash: 'e628c8fcb4bca3f3456640a8b56d1ae98c594e24'
   conf.gem github: 'mrbgems/mruby-yaml',        checksum_hash: '0606652a6e99d902cd3101cf2d757a7c0c37a7fd'
   conf.gem github: 'eagletmt/mruby-etc',        checksum_hash: 'v0.1.0'
+
+  conf.gem github: 'eagletmt/mitamae-secrets', checksum_hash: '959716b8054f144ba26ba5b3b247ba4ccb54d591'
+  conf.gem github: 'hanazuki/mitamae-apt'    , checksum_hash: '2ad774b626b557dd81c85d5968f3882d73dc84c5'
 end
 
 def debug_config(conf)
