From: Kasumi Hanazuki <kasumi@rollingapple.net>
Date: Sat, 10 Nov 2018 12:43:18 +0000
Subject: mrbgems

---
 build_config.rb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build_config.rb b/build_config.rb
index a1c6005..93d9bec 100644
--- a/build_config.rb
+++ b/build_config.rb
@@ -13,6 +13,7 @@ def gem_config(conf)
   conf.gem mgem: 'mruby-socket',         checksum_hash: 'a8b6d6ee4c6ccea81a805cc9204b36c3792123c9'
   conf.gem github: 'k0kubun/mruby-erb',  checksum_hash: '978257e478633542c440c9248e8cdf33c5ad2074'
   conf.gem github: 'eagletmt/mruby-etc', checksum_hash: 'v0.1.0'
+  conf.gem github: 'eagletmt/mitamae-secrets', checksum_hash: 'v0.1.1'
 end
 
 def debug_config(conf)
